src/generators/server_settings_generators/generate_all_generator.py

```diff
@@ -0,0 +1,59 @@
+from steamship import Task
+from steamship.agents.schema import AgentContext
+
+from generators.server_settings_generator import ServerSettingsGenerator
+from utils.agent_service import AgentService
+
+# Note: should_block is True for all of these because we don't want to awaken the parallel state management ghosts.
+GENERATE_KEY_PATHS_AND_SHOULD_BLOCK = [
+    [["narrative_voice"], True],  # Genre
+    [["narrative_tone"], True],  # Writing Style
+    [["name"], True],
+    [["short_description"], True],
+    [["description"], True],
+    [["adventure_goal"], True],
+    [["adventure_background"], True],
+    # [["image"], True],
```

Comment:
What's up with the commented images here?

Reply:
I wanted to get everything in and up & running for folks w/o images. Then do another pass to add them in.